blivet/devicelibs/dasd.py

```diff
@@ -144,6 +144,30 @@ def is_ldl_dasd(device):
         return False
     return True
 
+def is_fba_dasd(device):
+    """Is the device a FBA DASD?"""
+    if not device.startswith("dasd"):
+        # not a dasd; bail
+        return False
+
+    device = "/dev/%s" % (device,)
```

Comment:
The `is_fba_dasd` function is copy-pasted from `is_ldl_dasd`. I could change  `is_fba_dasd`, but then `is_ldl_dasd` should be changed as well. However, I would prefer not to do that since it is for RHEL and s390x is not so easy to test. 

Nevertheless, I can do that in a pull request for `2.1-devel` branch (https://github.com/rhinstaller/blivet/pull/552), since the `bd_s390_dasd_is_fba` function is already prepared for that (https://github.com/rhinstaller/libblockdev/pull/170). I should just call `blockdev.s390.dasd_is_fba(device.path)`, right?

What do you think?